signals/sender/sender_is_known_msft_domain copy.yml

```diff
@@ -0,0 +1,14 @@
+name: "Sender: Known MSFT domain"
+type: "query"
+source: |
```

Comment:
May want to add a DMARC check here, unless risk score will weigh a DMARC failure more/a failure would be accounted for elsewhere.